### PR TITLE
release(v0.12.0-alpha.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.12.0-alpha.1 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Review by rsmitty — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,350 @@
...
+### Support for Self-hosted Control Plane Dropped
+
+> **Note**: This item only applies to clusters bootstrapped with Talos <= 0.8.
+
+Talos 0.12 completely removes support for self-hosted Kubernetes control plane (bootkube-based).
+Talos 0.9 introduced support for Talos-managed control plane and provided migration path to convert self-hosted control plane
+to Talos-managed static pods.
+Automated and manual conversion process is available in Talos from 0.9.x to 0.11.x.
+For clusters bootstrapped with bootkube (Talos <= 0.8), please make sure control plane is converted to Talos-managed before
+before upgrading to Talos 0.12.
```
Comment (sergelogvinov) [inline CHANGELOG.md:20]: before - 2 times

Comment (smira) [reply CHANGELOG.md:20]: thanks, I'll get it fixed for the next beta!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,350 @@
...
+* Kubernetes: 1.22.0
+* containerd: 1.5.5
+* runc: 1.0.1
+* GRUB: 2.06
+* Talos is built with Go 1.16.7
+
+
+### Contributors
+
+* Andrey Smirnov
```
Comment (Ulexus) [inline CHANGELOG.md:97]: The really important contributors get to be listed twice.

Comment (smira) [reply CHANGELOG.md:97]: Changing email leads to duplicates )

**Pull request merged**